### PR TITLE
Meta styling

### PR DESCRIPTION
**Pull request by nicl**
## What does this change?

Adds meta info and styling for article body. Some of the data is still static as we don't currently send it to dotcom-rendering. So will have to remedy that in a future PR.

## Why?

MVP.

## Caveats/thoughts

There are a few questions here now that we have a bit more styling:

* should we collect some standard font styles somewhere (guui?).

It's not really a problem yet, but it feels like an area where re-use might be useful.

* validation of CAPI data and failure

At the moment we just default to empty string for pretty much everything here (in parse-capi), which allows more graceful failure but also doesn't really tell us when CAPI data is broken/not what we expect. It might be better to validate the model and immediately fail if the data is different from what we expect or required fields are missing.

* date formatting

I've used https://www.npmjs.com/package/dateformat which is lightweight (1.5kb gzipped vs 64kb for Moment.js which is crazy big). Seems to do the job, but worth flagging as we don't have a solution to this in frontend or here before now. JS Date object formatting sucks unfortunately.

screen shot 2018-08-21 at 14 25 27
screen shot 2018-08-21 at 14 26 01

### CODE REVIEW

File: frontend/components/ArticleBody.js [hunk 1]
```diff
@@ -158,7 +321,41 @@ const ArticleBody = ({ CAPI }: Props) => (
                     }}
                 />
             </div>
-            <div className={meta}>Meta</div>
+            <div className={meta}>
+                <div className={profile(pillarColour)}>{CAPI.author}</div>
+                <div className={twitterHandle}>
+                    {/* TODO - from the contributor type tag */}
+                    <TwitterIcon2 /> @ByRobDavies
```
Comment (SiAdcock) [inline frontend/components/ArticleBody.js]: Is there  a better name for this SVG?

File: frontend/components/ArticleBody.js [hunk 2]
```diff
@@ -141,14 +186,132 @@ const bodyStyle = css`
...
+
+    :hover {
+        background-color: ${colour};
+        border-color: ${colour};
+        fill: white;
+    }
+`;
+
+const ageWarning = colour => css`
+    font-size: 12px;
```
Comment (SiAdcock) [inline frontend/components/ArticleBody.js]: Font sizes are becoming a bit haphazard. Maybe we should package these up in pasteup somehow

File: frontend/components/ArticleBody.js [hunk 3]
```diff
@@ -1,7 +1,14 @@
 // @flow
 
 import { css } from 'react-emotion';
-import palette from '@guardian/pasteup/palette';
+import palette, { pillars } from '@guardian/pasteup/palette';
+import TwitterIcon from '@guardian/pasteup/icons/twitter.svg';
+import TwitterIcon2 from '@guardian/pasteup/icons/twitter-2.svg';
+import FbIcon from '@guardian/pasteup/icons/facebook.svg';
```
Comment (SiAdcock) [inline frontend/components/ArticleBody.js]: Can we go with the full FacebookIcon for consistency?

File: frontend/components/ArticleBody.js [hunk 4]
```diff
@@ -141,14 +186,132 @@ const bodyStyle = css`
...
+
+const shareIcons = css`
+    ${leftCol} {
+        border-bottom: 1px solid ${palette.neutral[5]};
+        padding-bottom: 6px;
+        margin-bottom: 6px;
+    }
+`;
+
+const share = colour => css`
```
Comment (SiAdcock) [inline frontend/components/ArticleBody.js]: Shall we call this shareIcon?

File: frontend/components/ArticleBody.js (hunk 2 above)
Comment (nicl) [reply frontend/components/ArticleBody.js]: Though I'd probably prefer that to be in a separate PR.

File: frontend/components/ArticleBody.js [hunk 5]
```diff
@@ -141,14 +186,132 @@ const bodyStyle = css`
     }
 `;
 
+const profile = colour => css`
```
[the remark this answers is not in the document]
Comment (SiAdcock) [reply frontend/components/ArticleBody.js]: This is probably a function of my frustration with how hard it is to debug Emotion. Let's park this for now and properly investigate the extent of Emotion's error handling

[the remark this answers is not in the document]
Comment (AWare) [reply frontend/components/ArticleBody.js]: I think that we should minimise the number of rules produced by dynamic css functions and then compose them with the static rules.

**Pull request merged**